boilerplate/_lib/container-make

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -x
```

Comment:
Do we still want this to have full debug output here? (was this me who left that in?)

Reply:
Yeah, because we're [sourcing common.sh](https://github.com/openshift/aws-account-operator/pull/547/files/b75a1b3840237c4d7fc58269a2c836e36d0fbbab#diff-9e781a63641c048df2cf2ee8dbc8b5cc6387b712eebb5c081ab424a1bf83abfcR10) you can switch on `-x` with an environment variable: `BOILERPLATE_SET_X=1`.

Reply:
> Yeah

By which I meant "Yeah, we can rip it out"